src/CTA.Rules.Metrics/MetricsTransformer.cs

```diff
@@ -121,6 +122,25 @@ internal static IEnumerable<MissingMetaReferenceMetric> TransformMissingMetaRefe
             return missingMetaReferenceMetrics;
         }
 
+        internal static IEnumerable<WebFormsActionMetric> TransformWebFormActionMetrics(MetricsContext context, ProjectResult projectResult)
+        {
+            var projectFile = projectResult.ProjectFile;
+            var webFormActionMetrics = new List<WebFormsActionMetric>();
+            foreach (var metric in projectResult.WebFormsMetricResults)
+            {
+                if(metric.ActionName == WebFormsActionType.FileConversion)
+                    webFormActionMetrics.Add(new FileConversionMetric(context, metric.ChildAction, projectFile));
+                else if(metric.ActionName == WebFormsActionType.ControlConversion)
+                    webFormActionMetrics.Add(new ControlConversionMetric(context,metric.ChildAction,metric.NodeName, projectFile));
+                else if(metric.ActionName == WebFormsActionType.ClassConversion)
+                    webFormActionMetrics.Add(new ClassConversionMetric(context, metric.ChildAction, projectFile));
+                else if(metric.ActionName == WebFormsActionType.DirectiveConversion)
+                    webFormActionMetrics.Add(new DirectiveConversionMetric(context, metric.ChildAction, projectFile));
```

Comment:
We should log a warning or error if ActionName does not match a WebFormsActionType